docs/content/concepts/cryptography/passkeys.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Passkeys
+description: Passkeys is a Sui primitive that enables you to sign-in to apps and send transactions to Sui using a private key securely stored on a authenticator. It uses the WebAuthn standard.
+---
+
+Passkeys provide a secure and user-friendly alternative for submitting transactions to Sui. Built on the **WebAuthn standard**, passkeys let users authenticate and sign transactions using:
+
+- Hardware security keys, such as YubiKeys)
+- Mobile devices, such as smartphones and tablets
+- Platform-based authenticators, such as Face ID and Touch ID
+
+Passkeys simplify authentication by removing the need to manage seed phrases or private keys manually. Instead, they rely on device-based authentication and cloud synchronization, allowing seamless, phishing-resistant access across multiple devices.
+
+By integrating passkeys, Sui improves security and accessibility, making it easier for users to manage their accounts without compromising decentralization or cryptographic security.
+
+Refer to [Typescript SDK support](https://sdk.mystenlabs.com/typescript/cryptography/passkey) on how to add passkey support to your application. Also feel free to refer to [SIP-9](https://github.com/sui-foundation/sips/blob/main/sips/sip-9.md) for product specification.
```

Comment:
nit: standardize the passkey usage, referring to it as passkey instead of passkeys (to be consistent with code)